core/scripts/legacy-javascript/create-polyfill-module-data.js

```diff
@@ -0,0 +1,286 @@
+  'es.typed-array.fill',
+  'es.typed-array.filter',
+  'es.typed-array.find',
+  'es.typed-array.find-index',
+  'es.typed-array.find-last',
+  'es.typed-array.find-last-index',
+  'es.typed-array.for-each',
+  'es.typed-array.from',
+  'es.typed-array.includes',
+  'es.typed-array.index-of',
+  'es.typed-array.iterator',
+  'es.typed-array.join',
+  'es.typed-array.last-index-of',
+  'es.typed-array.map',
+  'es.typed-array.of',
+  'es.typed-array.reduce',
+  'es.typed-array.reduce-right',
+  'es.typed-array.reverse',
+  'es.typed-array.set',
+  'es.typed-array.slice',
+  'es.typed-array.some',
+  'es.typed-array.sort',
+  'es.typed-array.subarray',
+  'es.typed-array.to-locale-string',
+  'es.typed-array.to-string',
+  'es.unescape',
+  'esnext.aggregate-error',
+  'esnext.global-this',
+  'esnext.typed-array.at',
+  'esnext.typed-array.find-last',
+  'esnext.typed-array.find-last-index',
+  'web.atob',
+  'web.btoa',
+  'web.dom-collections.for-each',
+  'web.dom-collections.iterator',
+  'web.dom-exception.constructor',
+  'web.dom-exception.to-string-tag',
+  'web.queue-microtask',
+  'web.self',
+  'web.timers',
+  'web.url',
+  'web.url.to-json',
+  'web.url-search-params',
+];
+
+const unneededSkips = modulesToSkip.filter(m => !polyfillsNotNeededForBaseline.list.includes(m));
+if (unneededSkips.length) {
+  console.error('Remove these skipped modules:');
+  console.error(unneededSkips);
+  process.exit(1);
+}
+
+/** @type {PolyfillModuleData} */
+const data = [
+  {name: 'focus-visible', modules: ['focus-visible']},
+  {name: 'Error.prototype.cause', modules: ['es.error.cause']},
+];
+
+for (const polyfillModuleName of polyfillsNotNeededForBaseline.list) {
+  // Skip if manualy defined above already.
+  if (data.some(d => d.modules.includes(polyfillModuleName))) {
+    continue;
+  }
+
+  if (modulesToSkip.includes(polyfillModuleName)) {
+    console.log(`skipping: ${polyfillModuleName}`);
+    continue;
+  }
+
+  const parts = polyfillModuleName.split('.').slice(1);
+
+  let className = kebabCaseToCamelCase(parts[0]);
+  className = className[0].toUpperCase() + className.slice(1);
+  if (parts[0] === 'json') className = 'JSON';
+  if (parts[0] === 'typed-array') className = 'TypedArray';
+  if (parts[0] === 'url') className = 'URL';
+
+  let prop = parts.length > 1 ? kebabCaseToCamelCase(parts[1]) : '';
+  if (parts[1] === 'to-json') prop = 'toJSON';
+
+  // @ts-expect-error
+  const maybeClassSymbol = global[className];
+
+  if (parts.length === 1) {
+    if (!maybeClassSymbol) {
+      throw new Error(polyfillModuleName);
+    }
+
+    data.push({name: className, modules: [polyfillModuleName], corejs: true});
+  } else {
+    try {
+      const instance = maybeClassSymbol();
+      if (instance[prop] !== undefined) {
+        // eslint-disable-next-line max-len
+        data.push({name: `${className}.prototype.${prop}`, modules: [polyfillModuleName], corejs: true});
+        continue;
+      }
+    } catch {}
```

Comment:
There must be a lot of classes that his this `catch` right? Or is it mostly `Array` and `Map` stuff that works fine with no constructor params?